src/main/webapp/app/exercises/text/assess-new/text-assessment-area/text-assessment-area.component.ts

```diff
@@ -27,11 +37,23 @@ export class TextAssessmentAreaComponent implements OnChanges {
     @Input() textBlockRefs: TextBlockRef[];
     @Output() textBlockRefsChange = new EventEmitter<TextBlockRef[]>();
     selectedRef: TextBlockRef | null = null;
+    wordCount = 0;
+    characterCount = 0;
+
+    constructor(private stringCountService: StringCountService) {}
 
     /**
      * Life cycle hook to indicate component change
      */
-    ngOnChanges(): void {
+    ngOnChanges(changes: { [propKey: string]: SimpleChange }): void {
+        for (const propName in changes) {
+            if (propName === 'submission') {
+                const changedSubmission = changes[propName].currentValue;
+                this.wordCount = this.stringCountService.countWords(get(changedSubmission, 'text'));
```

Comment:
The lodash `get` utility shouldn't be needed here.

Reply:
Removed  it in a337fc464e6df41d77a4d12374e7066b5c024882